Userland/Utilities/cal.cpp

```diff
@@ -68,7 +69,12 @@ enum class Header {
     Month,
 };
 
-static ErrorOr<Vector<String>> month_lines_to_print(Header header_mode, int month, int year)
+enum class StartsWeek {
```

Comment:
This name is a little clumsy, `StartOfWeek` would be better.

Reply:
(theoretically each day could start it… 🤔 not sure how interested weʼre in that! 😅)

Reply:
@kleinesfilmroellchen done

@konradekk `GNU cal` gives us that option with `--starts-week` - in my opinion it's pointless as I don't think anyone starts a week from anything else than Monday or Sunday. Correct me if I'm wrong though.

Reply:
well… see [the first sentence of last paragraph for wikipedia article on week](https://en.wikipedia.org/wiki/Week#:~:text=Cultures%20vary%20in%20which%20days%20of%20the%20week%20are%20designated%20the%20first%20and%20the%20last%2C%20though%20virtually%20all%20have%20Saturday%2C%20Sunday%20or%20Monday%20as%20the%20first%20day.) (a chrome/chromium/edge link).

not me to decide – just pointing out! 😅
(thereʼs a similar comment from @alimpfard about default value, i guess.)

Reply:
@konradekk Convinced by @alimpfard's great idea to implement reading this from Calendar settings as a default, so done.